Client/Frontend/Browser/HomePanel/NTPModels.swift

```diff
@@ -56,10 +56,19 @@ class NTPBackground: Codable {
         let y: CGFloat?
     }
     
-    lazy var image: UIImage? = {
+    var image: UIImage? {
         // Remote resources are downloaded files, so must be loaded differently
-        packaged == true ? UIImage(named: imageUrl) : UIImage(contentsOfFile: imageUrl)
-    }()
+        if packaged == true {
+            // Load without cache if possible
+            if let path = Bundle.main.path(forResource: imageUrl, ofType: nil) {
+                return UIImage(contentsOfFile: path)
+            }
```

Comment:
Is this needed? The previous solution didn't compute the path like this?

Reply:
Agreed `imageUrl` and path from `imageUrl` with nil type is same in this context.

Reply:
They are different. `UIImage(named:)` does internal caching in memory. `UIImage(contentsOfFile:)` does caching but will purge it from memory, and will reload it from file as necessary.